compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/SkiaParagraph.skiko.kt

```diff
@@ -729,8 +729,9 @@ internal class ParagraphBuilder(
         return null
     }
 
-    private fun textStyleToParagraphStyle(style: TextStyle): ParagraphStyle {
+    private fun textStyleToParagraphStyle(style: TextStyle, computedStyle: ComputedStyle): ParagraphStyle {
         val pStyle = ParagraphStyle()
+        pStyle.textStyle = makeSkTextStyle(computedStyle)
```

Comment:
Maybe we should cache this value to avoid extra JNI calls?

Reply:
Well, it does not make sense, `defaultStyle` is computed for each call

Reply:
`makeSkTextStyle` is already uses a cache inside.

> is computed for each call
Even if we create `defaultStlye` each call, we compare it by value (because it is a data class), not by reference.

Also the cache is probably not because of JNI calls. JNI call itself is a rare bottleneck.